tasks/mx/inegi.py

```diff
@@ -216,19 +221,22 @@ def columns(self):
             weight=0,
             targets={geom: GEOM_REF},
         )
-        name = OBSColumn(
-            type='Text',
-            weight=0,
-        )
+        cols = OrderedDict([('the_geom', geom),
+                            ('cvegeo', geom_ref)
+                        ])
+
+        if self.resolution in RESPROPNAMES:
+            cols['nomgeo'] = OBSColumn(
+                type='Text',
```

Comment:
This will result in overlapping IDs, because `id` is not specified separately for each possible source geom.